src/styles/stars.scss

```
@@ -0,0 +1,18 @@
+.stars {
+  display: flex;
+  padding-top: 10px;
+}
+
+.stars__item {
+  @include star-size;
+  margin-right: 4px;
+  background: center url(../images/star.svg) no-repeat;
+}
+
+@for $i from 1 through 5 {
+  .stars {
+    &--#{$i} :nth-child(-n + #{$i}) {
+      background-image: url(../images/star-active.svg);
+    }
+  }
+}
```

```suggestion
.stars {
  display: flex;
  padding-top: 10px;


&__item {
  @include star-size;
  margin-right: 4px;
  background: center url(../images/star.svg) no-repeat;
}

@for $i from 1 through 5 {

    &--#{$i} :nth-child(-n + #{$i}) {
      background-image: url(../images/star-active.svg);
    }
 
}
}
```
Сделай везде вложенность